spec/controllers/redirect/help_center_controller_spec.rb

```diff
@@ -42,7 +42,7 @@
         expect(response).to redirect_to redirect_url
         expect(@analytics).to have_logged_event(
           'External Redirect',
-          redirect_url: redirect_url,
+          include(redirect_url: redirect_url),
```

Comment:
this is the same as `hash_including` right? WDYT of using that instead for clarity?
```suggestion
          hash_including(redirect_url: redirect_url),
```

Reply:
Sure, yeah, I'll make that change